src/rest-server/src/models/vc.js

```diff
@@ -18,44 +18,35 @@
 
 // module dependencies
 const unirest = require('unirest');
+const xml2js = require('xml2js');
 const yarnConfig = require('../config/yarn');
 const createError = require('../util/error');
+const logger = require('../config/logger');
 
-class VirtualCluster {
-  constructor(name, next) {
-    this.getVcList((vcList, error) => {
-      if (error === null) {
-        if (name in vcList) {
-          for (let key of Object.keys(vcList[name])) {
-            this[key] = vcList[name][key];
-          }
-        } else {
-          error = createError('Not Found', 'NoVirtualClusterError', `Virtual cluster ${name} is not found.`);
-        }
-      }
-      next(this, error);
-    });
-  }
 
+class VirtualCluster {
   getCapacitySchedulerInfo(queueInfo) {
     let queues = {};
+
     function traverse(queueInfo, queueDict) {
       if (queueInfo.type === 'capacitySchedulerLeafQueueInfo') {
         queueDict[queueInfo.queueName] = {
-          capacity: queueInfo.absoluteCapacity,
-          maxCapacity: queueInfo.absoluteMaxCapacity,
+          capacity: parseInt(queueInfo.absoluteCapacity),
```

Comment:
use `Number()` instead of `parseInt()` to avoid number base issues.

Reply:
Resolve it by `math.round`

Reply:
Built-in [`Math.round`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round) object is enough